build/ci/pr/Jenkinsfile

```diff
@@ -52,33 +52,31 @@ pipeline {
                             notOnlyDocs()
                         }
                     }
-                    agent {
-                        label 'linux'
-                    }
                     steps {
                         createConfig()
-                        sh """
-                            cat >deployer-config.yml <<EOF
-id: gke-ci
-overrides:
-  kubernetesVersion: "1.12"
-  clusterName: $BUILD_TAG
-  vaultInfo:
-    address: $VAULT_ADDR
-    roleId: $VAULT_ROLE_ID
-    secretId: $VAULT_SECRET_ID
-  gke:
-    gCloudProject: $GCLOUD_PROJECT
-EOF
-                            make -C build/ci TARGET=ci-e2e ci
-                        """
+                        deployerConfig()
+                        sh 'make -C build/ci TARGET=ci-e2e ci'
                     }
                 }
             }
         }
     }
 
     post {
+        always {
```

Comment:
does it always run even if tests succeed?

Reply:
If I'm reading the CI output that looks to be the case. It looks like it is trying to access the cleaned up resources and failing, even though the tests themselves succeeded

Reply:
> does it always run even if tests succeed?

Right now yes, I did it just to check if it works. Because there is an issue that it can't be tested on local Jenkins https://github.com/elastic/infra/issues/13827. When it will work, I will do it to work only on test fail

Reply:
> Also it looks like it's included in the pr job, but would be mostly useful in the e2e job?

Sure, as I mentioned in previous PR I just did it for PR job inly to check if it works. When it works, adding this to e2e test jobs will take 5 min 😃